packages/lit-dev-content/site/docs/v3/data/context.md

```diff
@@ -28,7 +28,6 @@ Context definition (`logger-context.ts`):
 ```ts
 import {createContext} from '@lit/context';
 import type {Logger} from 'my-logging-library';
-export type {Logger} from 'my-logging-library';
```

Comment:
This line is actually not a duplicate and is meant to be a re-export of the type from this module so that consumer won't need to import it from `'my-logging-library'` package, and rather import it from this context module instead as shown in the consumer code example below.

Reply:
ah, right!
sorry, not sure how I overlooked that